core/lib/server/pub.ts

```diff
@@ -1095,13 +1113,10 @@ export const updatePub = async ({
 			values: pubValues,
 		});
 
-		const vals = Object.entries(processedVals).flatMap(([slug, value]) => ({
-			slug,
-			value,
-		}));
+		const normalizedValues = normalizePubValues(processedVals);
 
 		const pubValuesWithSchemaNameAndFieldId = await validatePubValues({
```

Comment:
I thought I wanted to update `updatePub` to also be able to take an array of related pubs, i.e. `slug: [{value: 'role', relatedPubId: 'xxx'}]` which `createPub` has. But now I'm not so sure, since this function has `includeRelations: false`, and even if I made it `true` updating the values on conflict gets an error `ERROR: ON CONFLICT DO UPDATE command cannot affect row a second time`

So I can create a pub with pub value `{slug: 'croccroc:author', value: '', relatedPubId: 'xxx'}`, but I can't update the pub values to add another pub value, i.e. `{slug: 'croccroc:author', value: '', relatedPubId: 'yyy'}`. I think this is because the insert into `pub_values` has to be more clever, maybe in the `onConflict` part??